Dockerfile

```diff
@@ -10,7 +10,7 @@ RUN apt-get update && \
         apt-get -y --no-install-recommends install aspell aspell-en
 
 RUN curl -sL https://cpanmin.us/ | perl - App::cpanminus && \
-        cpanm -nq App::cpm && \
+        cpanm -nq App::cpm Carton::Snapshot && \
```

Comment:
I *think* the base boxes we are using already have `cpanm` installed.  Just realized that recently: https://github.com/perl/docker-perl/blob/61f3aae15f583754767f4d5a969d609f1facdeb4/5.030.001-main-buster/Dockerfile#L22-L23

Maybe instead of running the `curl` command, we could run `cpanm --self-upgrade` before installing `cpm`.  Or add it to the `cpanfile`?

Reply:
I realize this is not the code you touched, but as long as we're making changes...

Reply:
ok got one idea 
`cpan --self-upgrade || curl ...`